includes/ziti/ziti.h

```diff
@@ -800,7 +811,36 @@ extern int ziti_close_write(ziti_connection conn);
 ZITI_FUNC
 extern int ziti_write(ziti_connection conn, uint8_t *data, size_t length, ziti_write_cb write_cb, void *write_ctx);
 
+/**
+ * @brief Bridge [ziti_connection] to a given IO stream
+ *
+ * This sets up the connection bridge: all bytes read from ziti_connection are forwarded to the IO stream, and vice a versa.
+ * Both ziti_connection and stream have to be established prior to this call.
+ *
+ * [on_close] is called after the bridge is terminated and ziti_connection was closed.
+ *
+ * @param conn
+ * @param stream
+ * @param on_close
+ * @return
+ */
+ZITI_FUNC
+extern int ziti_conn_bridge(ziti_connection conn, uv_stream_t *stream, uv_close_cb on_close);
 
+/**
+ * @brief Bridge [ziti_connection] to given IO file descriptors.
+ *
+ * All bytes read from ziti_connection are written to output fd, all bytes read from input fd are sent to ziti_connection.
+ *
+ * @param conn
+ * @param input
+ * @param output
+ * @param close_cb
+ * @param ctx
+ * @return
+ */
+ZITI_FUNC
+extern int ziti_conn_bridge_fds(ziti_connection conn, uv_os_fd_t input, uv_os_fd_t output, void (*close_cb)(void *ctx), void *ctx);
```

Comment:
why a different close_cb than on the stream version (and why doesn't the stream bridge include a void *ctx)?

Reply:
with the stream version we don't make any assumptions on how application want to close it -- so it is passed into the close callback. any custom data can be attached to that stream already `stream->data`